ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml

```diff
@@ -18,12 +18,10 @@ postsubmits:
         - --target=[output:stable:startup]
         - --target=[output:stable:sync]
         - --target=e2e-azure
-        - --template=/usr/local/e2e-azure
+        - --template=/usr/local/cluster-launch-e2e-azure.yaml
```

Comment:
We used this as a folder. Previously it was an issue with referring as a file. Need to test with mkpj